src/components/select/UISelect.vue

```diff
@@ -37,9 +37,7 @@
              @click="open ? closeList() : openList()">
             <template v-if="selectionCount > 0">
                 <slot name="selected" :selected="selected">
-                    <span :class="{ 'truncate': multi }">
-                        {{ selectionDisplay }}
-                    </span>
+                    {{ selectionDisplay }}
```

Comment:
So any exisitng multiselects will appear very differently now, potentially affecting the layout of any parent forms. Make sure to document this change in an upgrade guide on the release.

Reply:
In fact, might be a nice to start including an upgrade guide in the PR desc itself? Not extra work, just doing it before the merge, rather than at release, and should help the review process.

Reply:
The intention that the commits are somwhat describing what has changed.

Btw: This change was promted by a breaking styling with the previous implementation. When selecting more than there is width for the select still grew in it's width. (I checked som other component libraries and they just either listed all the selection or `this, that, +9 more`

Reply:
The commit messages are fine. What I'm saying is that release descriptions should absolutely have a detailed upgrade guide, so why not stick it here as well?

Reply:
Additionally, it means we can "pr" the upgrade guide, to ensure that it's easy to follow/descriptive enough. Maybe not so much for this change, but for things like the `main.ts` change.

Reply:
upgrade guide isn't a standard thing to have within the repo (like a `CHANGELOG.md`). Such thing would usually be between major version changes on a documentation site. However I see no issue having something on the PR description while in `0.x`